### ISSUE TITLE
Accessibility audit: (JAWS) /backlog Heading level order

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#310]
## Describe the issue
On /backlog the heading levels go 1,3,2.

## Steps to reproduce the issue
With VoiceOver on, bring up the rotor and arrow to the headings list.

<img width="642" alt="Screen Shot 2021-11-24 at 15 17 16" src="https://user-images.githubusercontent.com/809482/143272659-0a17d8c0-1f45-425b-8c82-07e89747510c.png">

## Actual vs expected behaviour
Headings should be nested and ranked in order.

## Fix recommendations
- [x] Change to heading level order to fix the page structure.
